unfollowtw.py

```diff
@@ -0,0 +1,68 @@
+import logging
+import os
+
+import tweepy
+
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
+
+def main():
+
+    consumer_key = os.environ.get('CONSUMER_KEY')
+    consumer_secret = os.environ.get('CONSUMER_SECRET')
+    access_token = os.environ.get('ACCESS_TOKEN')
+    access_token_secret = os.environ.get('ACCESS_TOKEN_SECRET')
+
+    auth = tweepy.OAuthHandler(consumer_key, consumer_secret)
+    auth.set_access_token(access_token, access_token_secret)
+
+    api = tweepy.API(
+        auth, wait_on_rate_limit=True,
+        wait_on_rate_limit_notify=True,
+        retry_count=10, retry_delay=5,
+        retry_errors=5
+    )
+
+    currrent_user = api.me().screen_name
+    friends = []
+    with open('friends_{}.txt'.format(currrent_user), 'r') as f:
+        for line in f:
+            friends.append(line.strip())
+
+    friends_cache = []
+    with open('friends_cache_{}.txt'.format(currrent_user), 'r') as f:
+        for line in f:
+            friends_cache.append(line.strip())
+
+    list_friends = [x for x in friends if x not in friends_cache]
+
+    with open('friends_cache_{}.txt'.format(currrent_user), 'a') as f:
+        try:
+            for fr in list_friends:
+                try:
+                    if api.get_user(fr).followers_count < 200:
+                        api.destroy_friendship(fr)
+                        logger.info("Unfollow %s", fr)
+                        f.write(fr + ' \n')
+                    else:
+                        logger.info("Add %s to file", fr)
+                        f.write(fr + ' \n')
+                except:
+                    logger.ERROR("Pass first try!")
+                    try:
+                        if len(api.followers_ids(fr)) < 200:
+                            api.destroy_friendship(fr)
+                            logger.info("Unfollow %s", fr)
+                            f.write(fr + ' \n')
+                        else:
+                            logger.info("Add %s to file", fr)
+                            f.write(fr + ' \n')
+                    except:
+                        logger.ERROR("Pass second try!")
```

Comment:
i think here will be better to use `logger.exception('')`, so you can get debug information after fail

Reply:
and in other cases of exception too

Reply:
Done